api/TrustedScript.json

```
@@ -46,8 +48,59 @@
           "deprecated": false
         }
       },
+      "toJSON": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TrustedScript/toJSON",
+          "spec_url": "https://w3c.github.io/webappsec-trusted-types/dist/spec/#dom-trustedscript-tojson",
+          "support": {
+            "chrome": {
+              "version_added": "90"
+            },
+            "chrome_android": {
+              "version_added": "90"
+            },
+            "edge": {
+              "version_added": false
```

```suggestion
              "version_added": "90"
```

Confirmed using https://mdn-bcd-collector.appspot.com/tests/api/TrustedScript/toJSON